app/main.py

```diff
@@ -2,14 +2,39 @@
 from typing import Optional
 from fastapi import FastAPI
 from starlette.middleware.cors import CORSMiddleware
+from app.api.v1.models.auth import AuthenticationCredential
+
 
 from app.api.v1.routes import get_routes
 from app.core.settings import Settings
 from app.core.scheduler import get_scheduler
 
+from app.api.v1.models import UserInfo, UserScope
+from app.core import AuthenticationContext, set_auth_credential, set_user_info
+
 logger = getLogger(__name__)
 
 
+def seed_default_user(settings: Settings = Settings()) -> None:
```

Comment:
this is not a great solution for seeding the database but it works for now. open to changing it.

Reply:
Could use secret env settings for the setup.  we can talk about it.